frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/ticketsTopPanel.component.tsx

```diff
@@ -28,9 +28,11 @@ import _ from 'lodash';
 import { BaseProperties, IssueProperties } from '../../tickets.constants';
 import { TitleProperty } from '../properties/titleProperty.component';
 import { PropertiesList } from '../propertiesList.component';
-import { IssuePropertiesInputs } from './IssuePropertiesInputs/issuePropertiesInputs.component';
-import { BaseTicketInfo, DescriptionProperty, TopPanel } from './ticketsTopPanel.styles';
+import { BaseTicketInfo, DescriptionProperty, TopPanel, FlexContainer } from './ticketsTopPanel.styles';
+import { IssuePropertiesInputs } from './issuePropertiesInputs/issuePropertiesInputs.component';
```

Comment:
I think the case of the directory is wrong. It starts with uppercase I
<img width="274" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/454e69f1-2b2a-42ee-a968-783b5d8b99c3">

I'd suggest to rename the directory, following the camelcase naming we currently have.